cmd/client.go

```diff
@@ -17,23 +19,75 @@ type Client struct {
 	mergeId        int
 	gitlabInstance string
 	authToken      string
+	logPath        string
+	debug          bool
 	git            *gitlab.Client
 }
 
-type Logger struct {
-	Active bool
+var requestLogger retryablehttp.RequestLogHook = func(l retryablehttp.Logger, r *http.Request, i int) {
+	logPath := os.Args[len(os.Args)-1]
+
+	file, err := os.OpenFile(logPath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	clonedRequest := r.Clone(r.Context())
+	var data []byte
+	reader := bufio.NewReader(clonedRequest.Body)
+	_, err = reader.Read(data)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Looks like this does not actually read data. I tried this:
```suggestion
	var data []byte
	if clonedRequest.Body != nil {
		data, err = io.ReadAll(clonedRequest.Body)
		if err != nil {
			panic(err)
		}
	}
```
and it reads data. Or we could consider https://pkg.go.dev/net/http/httputil#DumpRequest.